Dev 1.1

GigaZiv opened the pull request.

GigaZiv committed:
Dev 1.1

GigaZiv committed:
Dev 1.1

GigaZiv committed:
Dev 1.1

GigaZiv committed:
Dev 1.1

GigaZiv committed:
Dev 1.1

GigaZiv committed:
Dev 1.1

AleksandrKdev commented:
Приложение:
![image](https://github.com/user-attachments/assets/9f4c14e1-b231-4b4b-8520-f3e4af0f95dd)
Макет:
![image](https://github.com/user-attachments/assets/0a83e65d-588b-4154-a33e-3c44a8024e4b)

AleksandrKdev commented on app/src/main/res/layout/activity_main.xml:
```diff
@@ -1,19 +1,104 @@
 
-</androidx.constraintlayout.widget.ConstraintLayout>
\ No newline at end of file
+        <FrameLayout
```
Выглядит так, что тут framelayout лишний

AleksandrKdev commented on app/src/main/res/layout/activity_main.xml:
```diff
@@ -1,19 +1,104 @@
 <?xml version="1.0" encoding="utf-8"?>
-<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<LinearLayout
```
Этот linearlayout внутри содержит только один linearlayout, он выглядит лишним, можно попробовать от него избавиться

GigaZiv committed:
Dev 1.2

Внесено -
Цвета в векторных картинках тоже можно выносить в ресурсы, и писать @color/<цвет>

Исправлено -
Выглядит так, что тут framelayout лишний

Исправлено -
Этот linearlayout внутри содержит только один linearlayout, он выглядит лишним, можно попробовать от него избавиться
